src/store/global/global.actions.ts

```diff
@@ -443,7 +444,13 @@ function loadFiatExchangeRatesFailure(error: string): LoadFiatExchangeRatesFailu
   };
 }
 
-function openSnackbar(message: string, backgroundColor?: string): OpenSnackbar {
+function openSnackbar({
```

Comment:
All the actions across the app receive params instead of an object, I would keep it like that for consistency.

Reply:
Right. Done.